server/chat-plugins/modlog-viewer.ts

```diff
@@ -434,29 +434,34 @@ function buildResults(
 }
 
 async function getBattleSearch(
-	connection: Connection, userids: string[], turnLimit = 1,
-	month: string, tierid: ID
+	connection: Connection, userids: string[], month: string,
+	tierid: ID, turnLimit?: number
 ) {
 	userids = userids.map(toID);
 	const user = connection.user;
 	if (!user.can('forcewin')) return connection.popup(`/battlesearch - Access Denied`);
 
-	const response = await PM.query({userids, turnLimit, month, tierid});
-	connection.send(buildResults(response, userids as ID[], turnLimit, month, tierid));
+	const response = await PM.query({userids, turnLimit: turnLimit, month, tierid});
```

Comment:
Was this a typo?

```suggestion
	const response = await PM.query({userids, turnLimit, month, tierid});
```

Is something wrong with our CI? I swear this is something eslint checks...

Reply:
Was a typo, yeah.